tests/tests_all_options.yml

```
@@ -34,6 +36,22 @@
           set_fact:
             __ssh_is_ostree: "{{ __ostree_booted_stat.stat.exists }}"
 
+    - name: Enable installation of manual pages on Fedora/RHEL
+      ansible.builtin.lineinfile:
+        line: tsflags=nodocs
+        path: "{{ '/etc/dnf/dnf.conf'
+          if ansible_facts['distribution_version'] | int > 7
+          else '/etc/yum.conf' }}"
+        state: absent
+      when:
+        - ansible_facts['os_family'] == "RedHat"
+
+    - name: Reinstall manual pages for openssh-clients on RHEL
+      ansible.builtin.command: "{{ pkg_mgr }} reinstall -y openssh-clients"
+      when:
+        - ansible_facts['os_family'] == "RedHat" and not __ssh_is_ostree | bool
```

```suggestion
        - ansible_facts['os_family'] == "RedHat"
        - not __ssh_is_ostree | bool
```
since you are already using the multi-line when condition format

---

Thanks! Changed. I am never sure if it is `and` or `or` when writing the `when` condition ...